src/components/Card/Card.jsx

```diff
@@ -43,9 +44,14 @@ export const CardHeader = (
 ) => <div className="card--header">{children}</div>;
 
 export const CardTitle = (
-  { children, title } //eslint-disable-line
+  { children, title, handleTooltip } //eslint-disable-line
 ) => (
-  <span className="card--title" title={title}>
+  <span
+    className="card--title"
+    title={title}
+    onMouseEnter={() => handleTooltip(true)}
+    onMouseLeave={() => handleTooltip(false)}
```

Comment:
This overrides the default functionality of the tooltip, which breaks keyboard accessibility. The tooltip should be used directly and conditionally rendered if the text has been ellipsed.